### ISSUE TITLE
Create Ops ticket for sul-libsys-airflow-stage VM

### ISSUE THREAD

**Issue by jermnelson** [sul-dlss/libsys-airflow#285]
To reduce confusion and to be consistent with the deployed FOLIO environments, we need an Airflow Stage environment that replicates the existing `sul-libsys-airflow-dev` and `sul-libsys-airflow-prod` environments. (See https://github.com/sul-dlss/operations-tasks/issues/3145)

**Comment by jmartin-sul** [comment]
filed https://github.com/sul-dlss/operations-tasks/issues/3343

**Comment by jmartin-sul** [comment]
TODO: We'll want to spawn a new ticket for confirming this VM is setup properly, and doing an initial deployment of this codebase to it.  E.g. confirm that the puppet config for this new server is lined up with the current libsys-airflow prod config.  That ticket should also confirm that README instructions are correct for an initial deployment of libsys-ariflow (things may have drifted since they were initially written).

**Comment by jmartin-sul** [comment]
@lwrubel added passwords and such for this new env to vault under the `puppet/application/libsys_airflow/stage/` path.

i've put up a PR to add a cap config for deploying to the new stage environment (#307).

remaining TODO:
- [ ] get #307 reviewed and merged
- [ ] per @justinlittman in planning meeting yesterday, spawn a new ticket for confirming this VM is setup properly, by doing an initial deployment of this codebase to it (blocked by #307).  e.g. confirm that this new server is lined up with the current libsys-airflow prod config, and that the README instructions for initial deployment to the VM are correct (justin suspects the README may be a bit off)
